### ISSUE TITLE
Error on launch - Unexpected token '<', " <!DOCTYPE "... is not valid JSON

### ISSUE THREAD

**Issue by g-imconfused** [TheLastBen/fast-stable-diffusion#2911]
using A1111 in Colab, getting this error when I launch the UI today, worked fine last night. 

From console-

ERROR:    opening handshake failed
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/websockets/legacy/server.py", line 163, in handler
    await self.handshake(
  File "/usr/local/lib/python3.10/dist-packages/websockets/legacy/server.py", line 597, in handshake
    raise self.connection_closed_exc()  # pragma: no cover
websockets.exceptions.ConnectionClosedError: no close frame received or sent
2024-08-22 00:27:07 ERROR [uvicorn.error] opening handshake failed
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/websockets/legacy/server.py", line 163, in handler
    await self.handshake(
  File "/usr/local/lib/python3.10/dist-packages/websockets/legacy/server.py", line 597, in handshake
    raise self.connection_closed_exc()  # pragma: no cover
websockets.exceptions.ConnectionClosedError: no close frame received or sent
ERROR:    Exception in ASGI application
Traceback (most recent call last):

etc

**Comment by g-imconfused** [comment]
update - 

eventually got another error while tinkering with it, "OSError: [Errno 107] Transport endpoint is not connected" and found a better clue when googling that - https://github.com/TheLastBen/fast-stable-diffusion/issues/2111

TLDR; too many LoRAs makes SD sad in mysterious ways. I would never have figured this out from the errors that were thrown. Thanks, TheLastBen, always!
